content/articles/create-syna-blog/content.md

```diff
@@ -13,40 +13,45 @@ display_date = true
 
+Step by step:  
 ```bash
-git clone https://git.okkur.org/syna-start syna-blog
-cd syna-blog
-git submodule init
-git submodule update
+git clone --recursive https://git.okkur.org/syna-start my-blog
+cd my-blog
 hugo server -D
 ```
 
-Now you can open [http://localhost:1313](http://localhost:1313) and check out the sample website we just cloned. Take your time and check out the folder structure and run the server to browse the website. There are two pages in the repository, index and about. Each page has some fragments already displayed in them. You'll notice some fragments are the same. These fragments (nav, footer, copyright) are defined globally by their files in `content/_global` directory. If you want you can open `themes/syna/exampleSite` to check out our main website in depth.
+Now you can open [http://localhost:1313](http://localhost:1313) and check out the sample website we just cloned. Take your time and check out the folder structure and browse the website. There are two pages being created based on the starter repository, an index/main page and an about page. Each one has some fragments already being displayed. You'll notice some fragments are the same in both pages. These fragments (nav, footer, copyright) are defined globally and are shared by all pages, if not overwritten. These global fragments are defined within the `content/_global` directory.
 
-In order to create our blog, we need to clean up the sample project we just cloned and create the base files.
+In order to create our blog we need to add a blog directory for our section as well a few files:
 
 ```bash
-mkdir content/blog # our blog section
-touch content/blog/_index.md # blog's main page
+mkdir content/blog # our new blog section
+touch content/blog/_index.md # our blog's skeleton index/main page used for setting the title and description
 ```
 
-The `blog/_index.md` file is a "list" page in Hugo. If you're unfamiliar with how Hugo manages content please check out the [quick start](https://gohugo.io/getting-started/quick-start/) page of their documentation. A list page is configured by the `_index` directory in the same directory as the `_index.md` file. We will configure this page later on. Let's create our first blog post.
+The `blog/_index.md` declares the whole directory a section with child pages to Hugo. For more details on how Hugo manages content please check out the [quick start](https://gohugo.io/getting-started/quick-start/) page of their documentation.
+
+Let's first create our first blog post:
 
 ```bash
-mkdir content/blog/hello-world
-touch content/blog/hello-world/index.md
-touch content/blog/hello-world/content.md
+mkdir content/blog/hello-world # directory for our new blogpost will be served as a single page
+touch content/blog/hello-world/index.md # declaring the title and date of the whole page
```

Comment:
+1. These comments are really helpful to better understand the file and URL structures.